pkg/e2e/operators/ocmagent.go

```diff
@@ -2,218 +2,225 @@ package operators
 
 import (
 	"context"
-	"fmt"
-	"time"
+	"strings"
 
 	"github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	"github.com/openshift/osde2e/pkg/common/alert"
+	"github.com/openshift/osde2e/pkg/common/expect"
 	"github.com/openshift/osde2e/pkg/common/helper"
 	"github.com/openshift/osde2e/pkg/common/label"
-	"github.com/openshift/osde2e/pkg/common/util"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime/schema"
-	"k8s.io/apimachinery/pkg/util/wait"
-)
 
-var (
-	ocmAgentTestPrefix = "[Suite: informing] [OSD] OCM Agent Operator"
-	ocmAgentBasicTest  = ocmAgentTestPrefix + " Basic Test"
+	operatorhubv1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	networkingv1 "k8s.io/api/networking/v1"
+	"sigs.k8s.io/e2e-framework/klient/k8s"
+	"sigs.k8s.io/e2e-framework/klient/wait"
+	"sigs.k8s.io/e2e-framework/klient/wait/conditions"
 )
 
+var suiteName = "OCM Agent Operator"
```

Comment:
this is a global to the `operators` package so it overwrites anyone else using this. Can we create an `ocmagent` package in the `operators` directory to break this stuff out?

Reply:
Done!